test/verify/check-testlib

```diff
@@ -167,20 +167,22 @@ class TestRunTest(MachineCase):
         # create affected test for shell changes
         with open(__file__) as f:
             original_test = f.read()
-        affected_testfile = os.path.join(VERIFY_DIR, "test-shell-testlib")
-        writeFile(affected_testfile, original_test.replace("class NoTest", "class Test"))
+        affected_testfile = os.path.join(VERIFY_DIR, "check-shell-testlib")
+        new_content = original_test.replace("class Test", "class NeinTest")
+        writeFile(affected_testfile, new_content.replace("class NeinTestRetryExample", "class TestShell"))
+        os.chmod(affected_testfile, 0o555)
```

Comment:
Why does it do that? To prevent recursive editing?

Reply:
To add it +x